src/pages/workspaces/workspace/Dashboard/Dashboard.ts

```diff
@@ -0,0 +1,511 @@
+import { cond } from '@terra-ui-packages/core-utils';
+import _ from 'lodash/fp';
+import {
+  CSSProperties,
+  ForwardedRef,
+  Fragment,
+  ReactNode,
+  useCallback,
+  useEffect,
+  useImperativeHandle,
+  useMemo,
+  useState,
+} from 'react';
+import { div, h, i, span } from 'react-hyperscript-helpers';
+import * as breadcrumbs from 'src/components/breadcrumbs';
+import { ButtonPrimary, ButtonSecondary, Link, spinnerOverlay } from 'src/components/common';
+import { centeredSpinner, icon, spinner } from 'src/components/icons';
+import { InfoBox } from 'src/components/InfoBox';
+import { MarkdownEditor, MarkdownViewer } from 'src/components/markdown';
+import { SimpleTable } from 'src/components/table';
+import { WorkspaceTagSelect } from 'src/components/workspace-utils';
+import { Ajax } from 'src/libs/ajax';
+import { getEnabledBrand } from 'src/libs/brand-utils';
+import colors from 'src/libs/colors';
+import { reportError, withErrorReporting } from 'src/libs/error';
+import * as Nav from 'src/libs/nav';
+import { getLocalPref, setLocalPref } from 'src/libs/prefs';
+import { forwardRefWithName, useCancellation, useOnMount, useStore } from 'src/libs/react-utils';
+import { authStore } from 'src/libs/state';
+import * as Style from 'src/libs/style';
+import { append, formatBytes, newTabLinkProps, withBusyState } from 'src/libs/utils';
+import { canEditWorkspace, canWrite, isGoogleWorkspace, isOwner } from 'src/libs/workspace-utils';
+import SignIn from 'src/pages/SignIn';
+import { CloudInformation } from 'src/pages/workspaces/workspace/Dashboard/CloudInformation';
+import { DataUseLimitations, displayAttributeValue } from 'src/pages/workspaces/workspace/Dashboard/DataUseLimitations';
+import { OwnerNotice } from 'src/pages/workspaces/workspace/Dashboard/OwnerNotice';
+import { RightBoxSection } from 'src/pages/workspaces/workspace/Dashboard/RightBoxSection';
+import { WorkspaceInformation } from 'src/pages/workspaces/workspace/Dashboard/WorkspaceInformation';
+import { WorkspaceNotifications } from 'src/pages/workspaces/workspace/Dashboard/WorkspaceNotifications';
+import DashboardPublic from 'src/pages/workspaces/workspace/DashboardPublic';
+import { displayLibraryAttributes } from 'src/pages/workspaces/workspace/library-attributes';
+import { InitializedWorkspaceWrapper as Workspace, StorageDetails } from 'src/pages/workspaces/workspace/useWorkspace';
```

Comment:
I've seen some places where WorkspaceWrapper from libs/workspace-utils is aliased to Workspace. Here, InitializedWorkspaceWrapper is aliased to Workspace. I think we need to figure out a convention for workspace types to avoid confusion.